src/universal/components/SuggestMentionableUsers.js

```diff
@@ -0,0 +1,81 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {createFragmentContainer, graphql} from 'react-relay';
+import stringScore from 'string-score';
+import AsyncEditorSuggestions from 'universal/components/AsyncEditorSuggestions';
+import MentionUser from 'universal/components/MentionUser/MentionUser';
+import withAtmosphere from 'universal/decorators/withAtmosphere/withAtmosphere';
+
+const makeSuggestions = (triggerWord, teamMembers) => {
+  if (!triggerWord) {
+    return teamMembers.slice(0, 6);
+  }
+  return teamMembers.map((teamMember) => {
+    const score = stringScore(teamMember.preferredName, triggerWord);
+    return {
+      ...teamMember,
+      score
+    };
+  })
+    .sort((a, b) => a.score < b.score ? 1 : -1)
+    .slice(0, 6)
+    .filter((obj, idx, arr) => obj.score > 0 && arr[0].score - obj.score < 0.3);
```

Comment:
commented!
let's say you type "Foo". The options are "Food" and "Giraffe". Well, Giraffe has an F, so the score is > 0, but I'm _really_ sure you didn't mean giraffe because "Food" has a score of like .95.